packages/main/src/RPA/Salesforce.py

```diff
@@ -176,6 +217,12 @@ def _require_authentication(self) -> None:
         if self.sf is None:
             raise SalesforceAuthenticationError("Authentication is not completed")
 
+    def _require_no_session(self) -> None:
+        if self.session_id is not None or self.instance is not None:
```

Comment:
A simpler:
```suggestion
        if self.session_id or self.instance:
```
might work the same.